pkg/api/resourcegroup/v1alpha1/types.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package defines ResourceGroup schema.
+// Version: v1alpha1
+// swagger:meta
+package v1alpha1
+
+import (
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+const (
+	RGFileName       = "resourcegroup.yaml"
+	RGFileKind       = "ResourceGroup"
+	RGFileGroup      = "kpt.dev"
+	RGFileVersion    = "v1alpha1"
+	RGFileAPIVersion = RGFileGroup + "/" + RGFileVersion
+)
+
+// TypeMeta is the TypeMeta for ResourceGroup instances.
+var TypeMeta = yaml.ResourceMeta{
```

Comment:
TypeMeta could be misleading considering it is actually a `ResourceMeta` rather than a `TypeMeta`.  I'd suggest renaming to`DefaultMeta`

Reply:
I was replicating the naming convention used for the `Kptfile` types file, but using `DefaultMeta` makes more sense to me as well.